runtime/common/src/weights/mod.rs

```diff
@@ -28,6 +28,7 @@ pub mod pallet_multisig;
 pub mod pallet_preimage;
 pub mod pallet_scheduler;
 pub mod pallet_session;
+pub mod pallet_sudo;
```

Comment:
Is it a problem that this is here even when the mainnet build happens?

Reply:
if we are going to use the original weights then this is going to be redundant anyway.